### DISCUSSION TITLE
Two questions: Changing account email address and setting up custom email service

### DISCUSSION THREAD

**Discussion by andycwilliams** [emailjs-com/help#87] [Q&A]
Apologies if I missed some very obvious answers, but I'm trying to resolve a couple of issues.

The first is changing the email address that my account was registered with.

The second is whether it's possible to set up an email service for an straight webmail that was gotten from namecheap. Would that be through SMTP server?

**Comment by xr0master** [comment]
1. Write to support from the dashboard, the team will change your address for the user.
2. Yes, you can use the SMTP service to connect the Namecheap to the EmailJS
